src/main/java/in/projecteka/consentmanager/clients/DiscoveryServiceClient.java

```diff
@@ -33,4 +35,41 @@
                                 clientResponse -> Mono.error(ClientError.networkServiceCallFailed())))
                 .flatMap(responseSpec -> responseSpec.bodyToMono(PatientResponse.class));
     }
+
+    public Mono<PatientResponse> requestPatientFor(PatientRequest request, String url, String hipId) {
```

Comment:
If this is a call to gateway shouldn't it return void(and not PatientResponse)?

Reply:
returning Mono<Boolean> now. since zipping with delay in the next statement, returning Mono<Boolean>